src/main/java/nl/tudelft/contextproject/script/UpdateTask.java

```diff
@@ -0,0 +1,20 @@
+package nl.tudelft.contextproject.script;
+
+import nl.tudelft.contextproject.ContextTFP;
+
+import java.util.TimerTask;
+
+public class UpdateTask extends TimerTask {
```

Comment:
I think adding a lambda expression would compromise the readability and structure of the code. This way is fine by me, just plain and simple.